<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraditionalLoginWidget</class>
 <widget class="QWidget" name="TraditionalLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>realXtend Naali Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#TraditionalLoginWidget {
	background-color: none;
	/*background-color: qconicalgradient(cx:0, cy:1, angle:277.2, stop:0.267045 rgba(0, 169, 255, 255), stop:0.465909 rgba(0, 0, 0, 255));*/
}

QFrame#MainFrame {
	border: 0px;
	background-color: qlineargradient(spread:pad, x1:0.006, y1:0, x2:0, y2:1, stop:0.142045 rgba(229, 234, 243, 50), stop:0.295455 rgba(209, 215, 221, 110), stop:0.65 rgba(179, 183, 196, 110), stop:1 rgba(222, 227, 236, 100));
/* rgba(179, 183, 196, 100)*/
}

QGroupBox {
/*
	border: 2px solid;
	border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 169, 255, 118), stop:1 rgba(0, 0, 0, 0));
	border-radius: 0px;
	border-top-right-radius: 0px;
	border-bottom-left-radius: 0px;
	border-right: 0px;
	border-bottom: 0px;
*/
	border: 0px;
	margin-top:17px;
	margin-left: 0px;
	margin-right: 0px;
	font-size: 12px;
	font: bold;
	color: rgb(0, 0, 0);
 }

 QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left;
	padding: 0px 7px;
	background-color: rgba(0,0,0,0);
 }
/*
QLineEdit {
	border: 1px solid grey;
	border-radius: 0px;
	border-bottom-right-radius: 8px;
}

QLabel {
	color: rgb(229, 229, 229);
}

QLabel#label_OpenSim, QRadioButton#radioButton_realXtend {
	font-size: 12px;
}

QPushButton {
	color: rgb(0, 0, 0);
	min-width: 60px;
	font-size: 12px;
	border: 1px solid;
	border-color: rgb(48, 48, 48);
	border-radius: 8px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	padding: 2px 5px;
	background-color: qlineargradient(spread:pad, x1:0.488636, y1:0.273, x2:0.511, y2:1, stop:0 rgba(0, 0, 0, 50), stop:1 rgba(207, 207, 207, 100));
}

QPushButton:hover {
	border-color: rgba(0, 0, 0, 255);
	color: rgb(0,0,0);
	background-color: qlineargradient(spread:pad, x1:0.488636, y1:0.273, x2:0.511, y2:1, stop:0 rgba(0, 169, 255, 255), stop:1 rgba(0, 0, 0, 50));
}

QPushButton#pushButton_Connect, QPushButton#pushButton_Close {
	margin-right:15px;
}

*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_BottomContainer">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_LoginSplitHelp">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_LoginHolder">
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>15</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Authentication Type</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_TopRadioButtons">
                   <item>
                    <widget class="QRadioButton" name="radioButton_OpenSim">
                     <property name="styleSheet">
                      <string notr="true">/*color: rgb(229, 229, 229);*/</string>
                     </property>
                     <property name="text">
                      <string>OpenSim</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_realXtend">
                     <property name="styleSheet">
                      <string notr="true">/*color: rgb(229, 229, 229);*/</string>
                     </property>
                     <property name="text">
                      <string>realXtend</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Login Credentials</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_InputFields">
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="label_Username">
                       <property name="maximumSize">
                        <size>
                         <width>73</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Username</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="indent">
                        <number>0</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_Username">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>250</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>250</width>
                         <height>16777215</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="label_Password">
                       <property name="maximumSize">
                        <size>
                         <width>73</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Password</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="indent">
                        <number>0</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_Password">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>250</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>250</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_WorldAddress">
                       <property name="maximumSize">
                        <size>
                         <width>73</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>World address</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                       <property name="margin">
                        <number>0</number>
                       </property>
                       <property name="indent">
                        <number>0</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_WorldAddress">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>250</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>250</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
                     <item>
                      <widget class="QLabel" name="label_StartLocation">
                       <property name="text">
                        <string>Start location</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_StartLocation">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>250</width>
                         <height>0</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="demoWorldLabel">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Quick login to demo worlds</string>
              </property>
              <property name="indent">
               <number>7</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QFrame" name="DemoWorldFrame">
                <property name="styleSheet">
                 <string notr="true">QFrame#DemoWorldFrame {
	background-color: transparent;
	border: 0px;
	padding: 0px;
	margin: 0px;
	margin-left: 7px;
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="demoWorldsContainer"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_HelpHolder">
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>15</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font-size: 12px; font: bold;</string>
              </property>
              <property name="text">
               <string>Welcome to realXtend</string>
              </property>
              <property name="indent">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: transparent;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="plainText">
               <string>You are now using realXtends virtual world viewer called Naali. Next step is to get you into a world and for this we provide three types of login methods. 

OpenSim login requires you to have a existing account created into the server. Just insert your Firstname Last of the account to Username field, password and world address.

realXtend method brings you the ability to login to many worlds with the same avatar account. Again the account has to be created before doing a login. Next you need to know that the server supports realXtend login, basically this means the OpenSim server must be running our ModreX server component.

Third option is to login into realXtend Taiga and ScienceSim servers that are based on CableBeach services. Select the WebLogin tab from above and you should see a wiki page that provides you a list of open Taiga servers from the realXtend project and the community. And of course you can type the login url of the server to the Naali web browser manually. Web based authentications give you the possibility to login with your OpenID, Google and Yahoo accounts.</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
              <property name="backgroundVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame_BottomButtons">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="hlayout_buttons">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>1</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="statusLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Connect">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_ReturnToEther">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Return to Ether</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Exit">
               <property name="text">
                <string>Exit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
